src/modules/partition/core/PartitionCoreModule.cpp

```diff
@@ -982,10 +982,6 @@ PartitionCoreModule::layoutApply( Device* dev,
         {
             part->setLabel( "boot" );
         }
```

Comment:
The same principle applies for all partitions right? It so, we should also drop the `boot` hunk, correct?

Disclaimer: I know very little about the calamares internals :-P